runtime/doc/lsp.txt

```diff
@@ -1469,6 +1462,58 @@ save({lenses}, {bufnr}, {client_id})                 *vim.lsp.codelens.save()*
       • {client_id}  (integer)
 
 
+==============================================================================
+Lua module: vim.lsp.inlay_hint                                *lsp-inlay_hint*
+
+get({opts})                                         *vim.lsp.inlay_hint.get()*
+    Get the list of inlay hints, (optionally) restricted by buffer, client, or
+    range. This method does not make an LSP request, but just queries locally
+    stored inlay hints. The returned inlay hints are unresolved.
+
+    Example usage: >lua
+        local hint = vim.lsp.inlay_hint.get({ bufnr = bufnr })[1]
+        resolved_hint = client.request(methods.inlayHint_resolve, hint.inlay_hint, callback, bufnr)
+        vim.lsp.util.apply_text_edits(resolved_hint.textEdits, bufnr, 'utf-8')
+        command = resolved_hint.label[1].command
+        vim.lsp.buf.execute_command({ command = command })
+<
+
+    Parameters: ~
+      • {opts}  vim.lsp.inlay_hint.get.opts ? Optional table which holds the following optional fields:
```

Comment:
```suggestion
      • {opts}  vim.lsp.inlay_hint.get.opts? Optional table which holds the following optional fields:
```
Huh is the generator adding this space?

Reply:
I think so lol, I didn't add it. Unfortunately CI tends to fail when you deviate from the great generator

Reply:
Okay I promise I will look into it once I'm done with snippets.

Reply:
ground-up rewrite? :joy:

Reply:
Yeah. I want to blow it up with dynamite and write something _nice_.